Document model support (#75)
QGIS-Contribution/QGIS-ResourceSharing: _posts/2016-07-22-repository-structure.md

In order for the plugin to be able to read the repository correctly, the 

> repository must have a specific structure. Generally the structure looks like

Author: ```suggestion
repository must have a specific structure.
The general structure is shown in the illustration below.
Note that in the diagram, the ``models`` and ``rscripts`` folders are missing.
They are at the same level as ``image``, ``processing``, ``svg`` and ``symbol``.
```

> this (models and R scripts are missing, but are at the same level as *image*,

Reviewer: consider: "The structure follows the diagram below. Note that in the diagram the *models* and *rscripts* folders are missing. They are at the same level as *image*, *processing*, *svg* and *symbol*."

> this (models and R scripts are missing, but are at the same level as *image*,

Author: ```suggestion
```

> *processing*, *svg* and *symbol*):

Author: ```suggestion
```
